JSIL.Libraries/Includes/Bootstrap/Resources/Main.js

```diff
@@ -8,7 +8,7 @@ if (!$jsilcore)
 
 $jsilcore.getCurrentUICultureImpl = function () {
   var language;
-  var svc = JSIL.Host.getService("window", false);
+  var svc = JSIL.Host.getService("window", true);
```

Comment:
We need initialize current culture even in non-browser environment.
Really next line even tries to do it:
```
  if (svc) {
    language = svc.getNavigatorLanguage() || "en-US";
  } else {
    language = "en-US";
  }
```

But in non-browser environment it will not return null, but fail.
Now it will return null.